esphome/components/esp32/core.cpp

```diff
@@ -85,7 +91,7 @@ void loop_task(void *pv_params) {
 
 extern "C" void app_main() {
   esp32::setup_preferences();
-  xTaskCreate(loop_task, "loopTask", 8192, nullptr, 1, &loop_task_handle);
+  xTaskCreate(loop_task, "app_main", APP_MAIN_LOOP_STACK_SIZE, nullptr, 1, &loop_task_handle);
```

Comment:
```suggestion
  xTaskCreate(loop_task, "loopTask", APP_MAIN_LOOP_STACK_SIZE, nullptr, 1, &loop_task_handle);
```

This is potentially a breaking change. Please leave as is

Reply:
Could you explain how this is a breaking change?

As stated earlier there is no reference to "loopTask" in the entire workspace. Is there some tooling referring to it explicitly?

Reply:
We have tests that watch the logs for tasks and external components may match on loopTask